src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -130,19 +133,41 @@ export const TablesUserList: React.FC<TablesUserListProps> = ({
     [recentVenueUsers, user, venueName, videoRoom]
   );
 
+  const itemsToObjectByRefReducer = useCallback(
+    (obj: Record<string, WithId<User>[]>, table: Table) => ({
+      ...obj,
+      [table.reference]: recentVenueUsers.filter(
+        (user: User) =>
+          getUserExperience(venueName)(user)?.table === table.reference
+      ),
+    }),
+    [recentVenueUsers, venueName]
+  );
+
+  const usersSeatedAtTables = useMemo(
+    () => tables.reduce(itemsToObjectByRefReducer, {}),
+    [tables, itemsToObjectByRefReducer]
+  );
+
+  const isFullTable = useCallback(
+    (table: Table) => {
+      const numberOfSeatsLeft =
+        table.capacity &&
+        table.capacity - usersSeatedAtTables[table.reference].length;
+      return numberOfSeatsLeft === 0;
+    },
+    [usersSeatedAtTables]
+  );
+
   const tableLocked = useCallback(
     (table: string) => {
```

Comment:
`table` -> `tableReference` ?

Reply:
fixed in 46db62e1cd0e7d70ba136b3235574076c0ca5d6d